src/couch.js

```diff
@@ -12,23 +15,66 @@ async function _request(token, path, options) {
   return await response.json();
 }
 
+async function _requestAll(token, path, content) {
+  payload =
+    `{
+    "keys": [` +
+    content +
+    `]
+  }`;
+  let url = [couchUrl, path].join("/");
+  /* if (options) url = `${url}?${qs.stringify(options)}`; */
+
+  let response = await fetch(url, {
+    method: "POST",
+    body: payload,
+    headers: {
+      Authorization: `Bearer ${token}`,
+      Accept: "application/json"
+    }
+  });
+  return await response.json();
+}
 async function documents(token, db, options) {
   let result = await _request(token, [db, "_all_docs"].join("/"), options);
+  console.log("res", result);
   return result.rows.filter(row => !row.key.startsWith("_design"));
 }
 
+async function bulkId(token, db, metaIdList) {
+  console.log("insideBulk");
+  var contentarr = [];
+  contentarr = metaIdList.split("\n");
+  console.log("Split:-> ", contentarr);
+  var arrlength = contentarr.length;
+  content = "";
+  for (var index = 0; index < arrlength; index++) {
+    if (index != arrlength - 1) {
+      content = content + '"' + contentarr[index] + '",';
+    } else {
+      content = content + '"' + contentarr[index] + '"';
+    }
+    console.log("contentcheck-> ", content);
+  }
```

Comment:
Instead of formatting the list like this, just use JSON.stringify. You'll want the payload you pass to `_requestAll` to be a JSON object (or array, or whatever the Couch request takes) which gets stringified in the fetch call.